common/djangoapps/third_party_auth/models.py

```diff
@@ -673,6 +673,13 @@ def get_social_auth_uid(self, remote_id):
         """ Get social auth uid from remote id by prepending idp_slug to the remote id """
         return '{}:{}'.format(self.slug, remote_id)
 
+    def get_setting(self, name):
```

Comment:
A better name is `get_from_other_settings`?

Reply:
we have a method with same name(`get_setting`) for [OAuth2ProviderConfig](https://github.com/edx/edx-platform/blob/61e1eda20df2825a409db3e2d36c69d7c36d3e2d/common/djangoapps/third_party_auth/models.py#L402) with slightly different logic. used same name to avoid calling different method for OAuth2ProviderConfig object.